rtf

```diff
@@ -253,11 +253,11 @@ if Vagrant.has_plugin?("vagrant-cachier")
   config.cache.scope = :box
 end
 
-# Mac OS X workwaround
-config.vm.provision "fix-no-tty", type: "shell" do |s|
-    s.privileged = false
-    s.inline = "sudo sed -i '/tty/!s/mesg n/tty -s \\\\&\\\\& mesg n/' /root/.profile 2>/dev/null || true"
-end
+## Mac OS X workwaround
```

Comment:
Why commenting this ? This allow to use on mac os and should only be uncommented if this is the case ?